server/views/applications/pages/risk-to-self/vulnerability.njk

```diff
@@ -7,12 +7,34 @@
 
 {% block questions %}
 
-  {% if page.importDate %}
-    {% set date = page.importDate %}
-    {{dateOfOasysImport(date)}}
-  {% endif %}
+  <p class="govuk-body">
+    A person is vulnerable if there is a risk of other people harming them. Someone may be vulnerable in prison, in the community or both. 
+  </p>
 
-  {{
+  <p class="govuk-body">
+    Being vulnerable in prison includes being victimised, bullied or assaulted by others. 
+  </p>
+
+  <p class="govuk-body">
+    Being vulnerable in the community includes experiencing or being at risk of:
+    <ul class="govuk-list govuk-list--bullet">
+      <li>domestic abuse</li>
+      <li>physical, financial or psychological abuse</li>
+      <li>sex work</li>
+      <li>bullying</li>
+      <li>gang affiliated risks</li>
+      <li>cuckooing (their home being taken over and used for criminal activity)</li>
+      <li>modern day slavery, such as forced labour and sexual or criminal exploitation</li>
```

Comment:
are you content without a full stop at the end of the final bullet point?

Reply:
Yes all good, as per [Bullet points and steps section in the GOV.UK style guide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style):

> there is no full stop after the last bullet

Thanks for checking :)